src/components/button/Button.tsx

```diff
@@ -51,25 +51,32 @@ const getButtonVariables = ({
         iconMargin = 8;
     }
 
-    const buttonBg = alpha(mainColor, bgOpacity);
+    const buttonBg = setAlpha(mainColor, bgOpacity);
 
     const actionStyles = disabled
         ? null
         : css`
               &:hover {
                   background: ${disabled
                       ? buttonBg
-                      : blendColors(buttonBg, alpha(hoverColor, action.hoverOpacity))};
+                      : blendColors(buttonBg, setAlpha(hoverColor, action.hoverOpacity))};
               }
               &:active {
                   background: ${disabled
                       ? buttonBg
-                      : blendColors(buttonBg, alpha(hoverColor, action.activeOpacity))};
+                      : blendColors(buttonBg, setAlpha(hoverColor, action.activeOpacity))};
               }
               &:focus {
                   box-shadow: inset 0 0 0 2px rgba(255, 255, 255, 0.32),
                       inset 0 0 0 2px ${borderColor};
               }
+              &:focus:not(:focus-visible) {
+                  box-shadow: none;
```

Comment:
Um, so, I forgot about the secondary button also using the box-shadow for a border, this line now entirely removes the border when clicking on a secondary button. Should be an easy enough fix, just need to determine if the button variant is secondary and use the secondary button's border box-shadow instead of none.